src/Evolution/Ringdown/MinimumAhCExcisionRadius.cpp

```diff
@@ -0,0 +1,344 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+#include "Evolution/Ringdown/MinimumAhCExcisionRadius.hpp"
+
+#include <array>
+#include <cstddef>
+#include <optional>
+#include <vector>
+
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/Tags/TempTensor.hpp"
+#include "DataStructures/Tensor/IndexType.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/BlockLogicalCoordinates.hpp"
+#include "Domain/CoordinateMaps/Distribution.hpp"
+#include "Domain/CoordsToDifferentFrame.hpp"
+#include "Domain/Creators/BinaryCompactObject.hpp"
+#include "Domain/Creators/Sphere.hpp"
+#include "Domain/Creators/TimeDependentOptions/BinaryCompactObject.hpp"
+#include "Domain/Creators/TimeDependentOptions/ExpansionMap.hpp"
+#include "Domain/Creators/TimeDependentOptions/RotationMap.hpp"
+#include "Domain/Creators/TimeDependentOptions/ShapeMap.hpp"
+#include "Domain/Creators/TimeDependentOptions/Sphere.hpp"
+#include "Domain/StrahlkorperTransformations.hpp"
+#include "IO/H5/Dat.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/VolumeData.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/IO/ReadSurfaceYlm.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/Strahlkorper.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/StrahlkorperFunctions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/Serialization/Serialize.hpp"
+
+namespace {
+void get_grid_point(
+    const BlockLogicalCoords<3>& block_logical_coord,
+    const Domain<3>& ringdown_domain,
+    const domain::FunctionsOfTimeMap& ringdown_functions_of_time,
+    const tnsr::I<double, 3, Frame::Inertial>& exc_inertial_frame_point,
+    tnsr::I<double, 3, Frame::Grid>& exc_grid_frame_point, double match_time) {
+  // If the point is mapped to a block then the ringdown excision radius
+  // chosen does not enclose excisions A/B from the inspiral
```

Comment:
Because the points tested here are the excision boundaries? Please clarify this comment. Also maybe move it elsewhere because AFAICT this doesn't matter for this function.